### DISCUSSION TITLE
Flow and Speed settings after print is finished

### DISCUSSION THREAD

**Discussion by kisslorand** [bigtreetech/BIGTREETECH-TouchScreenFirmware#1406] [Ideas]
I am curious what others think about these settings. Should they remain at set values or should they reset to default after each print?
My personal opinion is that they should reset to default after each finished print.

**Comment by KiingYS** [comment]
Yeah, those two settings have to be reset to default. Usually when the user changes them, it's just to get a good adhesion on first layer or to print critical parts that requires slow extrusion. I think that there is a glitch on the filament "auto load". After it's loaded, a pop up appears saying "paused for user  continue?" when you click ok, you can't get rid of the pop up, you have to restart the printer
![20201224_174426](https://user-images.githubusercontent.com/60795090/103108066-35267300-4612-11eb-95c0-658f80fb0c8e.jpg)

**Reply by 3dfixer** [reply]
on the pause problem, its not bringing up the purge more/continue popup. to get out of it turn knob a few clicks and then push it. after a few seconds off it goes. if it purges more turn knob other way

**Comment by kisslorand** [comment]
If there's an issue, you should open a ticket about it. It's he right place to post to get it fixed.

**Comment by sarvenn** [comment]
It should reset to default as 100%.
I've added M220 S100 and M221 S100 to my both start and end gcodes for that reason @kisslorand . As you know when you turn the knob in tft mode it may change the feedrate unintentionally. That's why I find this as a solution to that

**Comment by kisslorand** [comment]
I made PR #1556 that resets them to 100% but @bigtreetech opted not to merge it.

**Reply by sarvenn** [reply]
Will I have a problem at the print statistics pop up at the end of the print? Should I delete M220/M221 commands from my endgcode? What do you think my friend?

**Comment by kisslorand** [comment]
Better set them at start, not at the end.
